src/generators/SpirvShaderGenerator.cpp

```diff
@@ -58,7 +64,11 @@ void CSpirvShaderGenerator::Generate()
 	m_uint4TypeId = AllocateId();
 	m_matrix44TypeId = AllocateId();
 	m_uintArrayTypeId = AllocateId();
```

Comment:
Probably need to allocate an id for the other array types (ie.: `m_uint8ArrayTypeId` & `m_uint16ArrayTypeId`) here.